crypto/rand/rand_lib.c

```diff
@@ -178,6 +178,8 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
                                    prediction_resistance,
                                    NULL, 0) != 0)
                 bytes = bytes_needed;
+            drbg->reseed_next_counter
+                = tsan_load(&drbg->parent->reseed_prop_counter);
```

Comment:
This is a default call back. I'm not sure if we should be doing this here. Do we support custom callbacks where they still use the parent?

Reply:
Wouldn't know how to implement custom callback functions, when the RAND_DRBG structure
is only defined by the local include crypto/rand/rand_lcl.h, no access functions exist.

Reply:
My point is a little that this now uses something in the rand_lcl.h file while it didn't before, and so an other callback can not do this.

Reply:
Well, I think when a non-default callback is used it will not work with the case parent != NULL,
but it might work with the case parent == NULL.
I wonder if the parent should be checked to be == NULL in RAND_DRBG_set_callbacks.

Reply:
Okay, I moved the case where parent == NULL to the caller.
And check that parent == NULL in the RAND_DRBG_set_callbacks.